feat!(*): use required name attribute instead of block label (#6384)
grafana/agent: docs/sources/flow/reference/components/prometheus.exporter.snmp.md

### target block

The `target` block defines an individual SNMP target.

> The `target` block may be specified multiple times to define multiple targets. `name` attribute is required and will be used in the target's `job` label.

Reviewer: ```suggestion
You can specify the `target` block multiple times to define multiple targets. The `name` attribute is required and is used in the target's `job` label.
```
